src/services/boardingHouse.ts

```
@@ -0,0 +1,138 @@
+		client: ApolloClient<any>,
+		roomId: string
+	): Promise<string> => {
+		const {data} = await client.mutate<{ endBoardingRoomContract: string }>({
+			mutation: BoardingHouseMutations.END_BOARDING_ROOM_CONTRACT,
+			variables: {roomId},
+		});
+		return data?.endBoardingRoomContract ?? "";
+	},
+	
+	patchBoardingRoom: async (
+		client: ApolloClient<any>,
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)